clients/web/test/spec/userSpec.js

```diff
@@ -96,6 +96,16 @@ describe('Create an admin and non-admin user', function () {
         }, 'front page to display');
     });
 
+    it('check the user count with an admin user', function () {
+        girderTest.login('admin', 'Admin', 'Admin', 'adminpassword!')();
+        girderTest.goToUsersPage()();
+        runs(function () {
+            expect($('span.g-user-count')).toBeDefined();
```

Comment:
This assertion will always pass, because even empty jQuery selectors are still truthy objects.

The next line will fail if the element doesn't exist, so you can remove this line entirely.